<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VENTANA</class>
 <widget class="QMainWindow" name="VENTANA">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>651</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Codigo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre Producto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Descripción</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>470</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cantidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P.Unitario</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_int">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>761</width>
      <height>321</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>651</width>
      <height>192</height>
     </size>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>124</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Codigo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre Producto                        </string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descripción</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P.Unitario</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>-</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="5" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="5" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="6" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="6" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="7" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="7" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="8" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="8" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="9" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="9" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bot_agregar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cod_bus">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ffff</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>21</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>320</y>
      <width>20</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="cant">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="des">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="precio">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="modificar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eliminar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>550</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
